experiments/context_gating/configs/experiment/context_encoder_concat.yaml

```diff
@@ -0,0 +1,16 @@
+# @package _global_
+group: context_encoder_concat
```

Comment:
- [ ] could you rename this group to `context_encoder` (and also the file)? currently we don't have any other way than concatenating